app/models/user.rb

```diff
@@ -228,6 +228,18 @@ def mfa_api_authorized?(otp)
     otp_verified?(otp)
   end
 
+  def mfa_required?
+    return false if mfa_enabled?
+    owner_of_most_downloaded_gem?
+  end
+
+  # NOTE:
+  # Do we want to differentiate the concept of mfa required for an acct where mfa is not enabled vs.
+  # Identifying which accounts require mfa, regardless whether it's enabled or not
+  def owner_of_most_downloaded_gem?
+    (rubygems & Rubygem.by_downloads.limit(Rubygem::MOST_DOWNLOADED)).any?
```

Comment:
Ashley and I had a quick convo about how we want the determine who is required to enable MFA. Right now it is dynamic, this raises some benefits and concerns:

- What if gem 101 becomes 100 when we release this policy? It's good that now they are required to enable MFA but if they become 100, they don't have any transition time when this hits. gem 100 becomes gem 101 and now they can disable MFA which is also weird.
- The query time of this method could slow these requests down a bit.

Here are the pros and cons for a static list:
- Can't enforce gems that surge in popularity during the policy period
- Is it PII to commit a list of gems? How are we storing this list?
- Faster to check a static list
- We know what gem accounts we are targeting and there are no surprises

With the reasons above, I'm leaning towards the static list.